src/riak_kv_ts_api.erl

```diff
@@ -184,8 +196,54 @@ build_object(Bucket, Mod, DDL, Row, PK) ->
     RObj = riak_object:newts(
              Bucket, PK, Obj,
              dict:from_list([{?MD_DDL_VERSION, ?DDL_VERSION}])),
-    {RObj, LK}.
-
+    {LK, RObj}.
```

Comment:
Yes, corresponds to line 235 in this file and `riak_kv_w1c_worker:ts_batch_put/7` (note the spec). Pretty sure I had it reversed in an earlier version of the code.